uikit-dz/uikit-dz/Cells/PostTableViewCell.swift

```diff
@@ -0,0 +1,303 @@
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.showsVerticalScrollIndicator = false
+        scrollView.showsHorizontalScrollIndicator = false
+        scrollView.isPagingEnabled = true
+        scrollView.delegate = self
+        return scrollView
+    }()
+    
+    private lazy var heartButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImage(named: Constant.heartPost.rawValue), for: .normal)
+        button.addTarget(self, action: #selector(buttonPressed), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var messageButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImage(named: Constant.messagePost.rawValue), for: .normal)
+        button.addTarget(self, action: #selector(buttonPressed), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var telegramButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImage(named: Constant.telegramPost.rawValue), for: .normal)
+        button.addTarget(self, action: #selector(buttonPressed), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var bookmarkButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImage(named: Constant.bookmarkPost.rawValue), for: .normal)
+        button.addTarget(self, action: #selector(buttonPressed), for: .touchUpInside)
+        return button
+    }()
+
+    // MARK: - Initializer
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupUI()
+        setupConstraints()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+
+    // MARK: - Private methods
+
+    private func setupUI() {
+        addSubview(profileImageView)
+        addSubview(profileTitleLabel)
+        addSubview(editButton)
+        addSubview(myscrollView)
+        addSubview(heartButton)
+        addSubview(messageButton)
+        addSubview(telegramButton)
+        addSubview(bookmarkButton)
+        addSubview(likesLabel)
+        addSubview(postLabel)
+        addSubview(profileCommentsImageView)
+        addSubview(commentsLabel)
+        addSubview(watchPostTimeLabel)
+        contentView.addSubview(customPageControl)
+        contentView.addSubview(myscrollView)
+        myscrollView.addSubview(mountainsImageView)
+        myscrollView.addSubview(mountainsImageView2)
+        myscrollView.addSubview(mountainsImageView3)
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
```

Comment:
```suggestion
        NSLayoutConstraint.activate([
```
разнеси эту простыню по методам